warehouse/templates/manage/account.html

```diff
@@ -29,32 +29,43 @@
 {% macro email_verification_label(email) -%}
 {% if email.verified %}
   {% if email.transient_bounces %}
-  {% set message = "Intermittent delivery problems may lead to verification loss" %}
-  <span class="badge badge--warning tooltipped tooltipped-s" data-original-label="{{ message }}" aria-label="{{ message }}"><i class="fa fa-exclamation-triangle"></i> Verified</span>
+  <span class="badge badge--warning">
+    <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+    Verified*
+  </span>
+  <span class="table__status-detail">*Intermittent delivery problems may lead to verification loss</span>
   {% else %}
   <span class="badge badge--success">
     <i class="fa fa-check" aria-hidden="true"></i>
     Verified
   </span>
   {% endif %}
-  {% else %}
-  {% set icon = 'exclamation-triangle' %}
+{% else %}
   {% if email.unverify_reason.value == "spam complaint" %}
-    {% set message = "Email from PyPI being treated as spam" %}
-    {% set icon = 'ban' %}
+  <span class="badge badge--danger">
+    <i class="fa fa-ban" aria-hidden="true"></i>
+    Unverified*
```

Comment:
It's a good idea - but I imagine that the frequency of some of these cases is so low that adding additional help text would only serve to clutter the FAQ (which is getting long as it is!)

Also, exposing the information directly, rather than requiring the user to search for it (even if it is just one click) is much more user friendly.